scripts/depends.sh

```diff
@@ -5,5 +5,5 @@ set -e
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 cd ${DIR}
 
-which bundler &>/dev/null || sudo gem install bundler
+which bundle &>/dev/null || sudo gem install bundler
```

Comment:
That's confusing because the name of the command is `bundle`, but the name of the package is `bundler`...

Reply:
exactly!